rosbag2_cpp/include/rosbag2_cpp/reader_interfaces/base_reader_interface.hpp

```diff
@@ -57,6 +58,8 @@ class ROSBAG2_CPP_PUBLIC BaseReaderInterface
   virtual void reset_filter() = 0;
 
   virtual void seek(const rcutils_time_point_value_t & timestamp) = 0;
+
+  virtual void add_event_callbacks(bag_events::ReaderEventCallbacks & callbacks) = 0;
```

Comment:
IMO. it will be much safer if `callbacks` parameter will be a const references.

Reply:
Done in db2fc1d.